manage.py

```diff
@@ -1,12 +1,22 @@
 #!/usr/bin/env python
-# -*- coding: utf-8 -*-
-from __future__ import unicode_literals, absolute_import
-
+"""Django's command-line utility for administrative tasks."""
 import os
 import sys
 
-if __name__ == "__main__":
+
+def main():
+    """Run administrative tasks."""
     os.environ.setdefault("DJANGO_SETTINGS_MODULE", "tests.settings")
-    from django.core.management import execute_from_command_line
+    try:
+        from django.core.management import execute_from_command_line
+    except ImportError as exc:
+        raise ImportError(
+            "Couldn't import Django. Are you sure it's installed and "
+            "available on your PYTHONPATH environment variable? Did you "
+            "forget to activate a virtual environment?"
+        ) from exc
+    execute_from_command_line(sys.argv)
 
-    execute_from_command_line(sys.argv)
\ No newline at end of file
+
+if __name__ == "__main__":
+    main()
```

Comment:
Do we need to keep this manage.py?

Reply:
In django-virtual-models, I have a separate example folder: https://github.com/vintasoftware/django-virtual-models

Reply:
I'll check

Reply:
I believe folks did it like that because of the need to add `celerybeat_status` to `INSTALLED_APPS`.

Reply:
Would renaming the `tests` folder to `example` be a middle ground?

Reply:
But I don't think Django Apps need manage.py. I see this project has one, but I think it's to run tests: https://github.com/django-extensions/django-extensions

Reply:
Yes, it's for testing and running the example

Reply:
Considering the [django-extensions](https://github.com/django-extensions/django-extensions), maybe having the test folder is a pattern?

Reply:
As for the `tests` folder, it seems to be a pattern indeed, I think we should keep it as it is in this case.

Other django apps:

- https://github.com/encode/django-rest-framework
- https://github.com/adamchainz/django-cors-headers
- https://github.com/graphql-python/graphene-django

Reply:
I'm trying to find an example with the `manage.py`, but so far I believe we have it so we can run and render the admin UI for testing/QA purposes.

Reply:
Having a tests folder is fine, but that's where the tests are, not the examples.
The idea for having a tests dir is to not distribute it along with the main app (it's not installed via pip for example)

Reply:
I see, but the `tests` folders have auxiliary files besides the `test_*` ones. For example:
-  `django-rest-framework` has [`models.py`](https://github.com/encode/django-rest-framework/blob/master/tests/models.py) and [`urls.py`](https://github.com/encode/django-rest-framework/blob/master/tests/urls.py).
- `django-cors-headers` has [`settings.py`](https://github.com/adamchainz/django-cors-headers/blob/main/tests/settings.py) and [`urls.py`](https://github.com/adamchainz/django-cors-headers/blob/main/tests/urls.py).

We would be doing something similar here

Reply:
yes, that's fine

Reply:
Django own code also does this.